src/curated-corpus/pages/SchedulePage/SchedulePage.tsx

```diff
@@ -523,13 +524,16 @@ export const SchedulePage: React.FC = (): ReactElement => {
       showNotification('Cannot schedule item without topic', 'error');
       return;
     }
-
+    const scheduledSource =
+      approvedItem.source === CorpusItemSource.Ml
+        ? ScheduledItemSource.Ml
+        : ScheduledItemSource.Manual;
     // Set out all the variables we need to pass to the mutation
     const variables = {
       approvedItemExternalId: approvedItem?.externalId,
       scheduledSurfaceGuid: values.scheduledSurfaceGuid,
       scheduledDate: values.scheduledDate.toISODate(),
-      source: ScheduledItemSource.Manual,
+      source: scheduledSource,
```

Comment:
I'm assuming this is called when a curator manually adds a URL on the schedule page.

I would keep the source set to Manual, because the item was manually scheduled, even if ML previously added the item to the corpus.
```suggestion
      source: ScheduledItemSource.Manual,
```